visualization/Readme.md

```diff
@@ -0,0 +1,32 @@
+This is a tool for data analysis. It has two pages: "Run model" and "Analyze".
+## Run Prompt page:
+This page allows for the analysis of model answers depending on different parameters we set up. It has three modes: "Complete", "Run one test", and "Run whole dataset".
+**Complete** mode allows for a conversation with the model. It has the minimum parameters to set up.
+**Run one test** mode allows you to send a single question to the model. It can be a question from the dataset (parameters `data_file` or `dataset` and `split_name` determine the dataset) or your custom question. The answer is also validated by comparing it with the `expected_answer` field.
+**Run whole dataset** mode allows you to launch the model with chosen parameters on the whole dataset. It will save results in an `output_file` and `save_metrics` paths if specified, or by default paths: `results/output-greedy.jsonl` and `results/metrics-greedy.jsonl`. If the "use random seed range" flag is turned on, the dataset will be launched for random seeds in the range from `start_random_seed` to `random_seed`. All results can then be analyzed on the "Analyze" page (you do not need to relaunch the program), and the parameters you used to launch the dataset can be found in the `results/parameters.jsonl` file and on the "Analyze" page as well.
```

Comment:
Let's just save metrics/outputs to one folder, so that you only need to specify folder name?

Reply:
Should it be `end_random_seed`?